libraries/botbuilder-core/src/testAdapter.ts

```diff
@@ -170,36 +287,13 @@ export class TestAdapter extends BotAdapter implements ExtendedUserTokenProvider
     }
 
     /**
-     * @private
-     * INTERNAL: called by a `TestFlow` instance to simulate a user sending a message to the bot.
-     * This will cause the adapters middleware pipe to be run and it's logic to be called.
-     * @param activity Text or activity from user. The current conversation reference [template](#template) will be merged the passed in activity to properly address the activity. Fields specified in the activity override fields in the template.
-     */
-    public receiveActivity(activity: string | Partial<Activity>): Promise<void> {
```

Comment:
>receiveActivity [](start = 11, length = 15)

This could be a breaking change. Others should weigh in.

Reply:
alias `receiveActivity` to `processActivity` to keep backward compatibility.